packages/inference/src/types.ts

```diff
@@ -33,6 +33,11 @@ export interface Options {
 	 * Requests can only be billed to an organization the user is a member of, and which has subscribed to Enterprise Hub.
 	 */
 	billTo?: string;
+
+	/**
+	 * Optional: override the default router.huggingface.co Inference Providers endpoint with any baseURL.
+	 */
+	endpointUrl?: string;
```

Comment:
I don't think that works without changes to `makeRequestOptions`, `endpointUrl` is in the `Args` not in the `Options`

Reply:
good catch => 5d352ac8d86ef2846f371ee3f80995bf353af1a3